src/patternfly/components/DataToolbar/data-toolbar.scss

```diff
@@ -0,0 +1,356 @@
+$pf-c-data-toolbar--spacer-map: build-spacer-map("none", "sm", "md", "lg");
+
+// Data toolbar
+.pf-c-data-toolbar {
+  --pf-c-data-toolbar--BackgroundColor: var(--pf-global--BackgroundColor--100);
+
+  // Content
+  --pf-c-data-toolbar__content--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-data-toolbar__content--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-data-toolbar__content--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-data-toolbar__content--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-data-toolbar__content--md--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-data-toolbar__content--md--PaddingLeft: var(--pf-global--spacer--md);
+
+  @media screen and (max-width: $pf-global--breakpoint--md) {
+    --pf-c-data-toolbar__content--PaddingRight: var(--pf-c-data-toolbar__content--md--PaddingRight);
+    --pf-c-data-toolbar__content--PaddingLeft: var(--pf-c-data-toolbar__content--md--PaddingLeft);
+  }
+
+  // Expandable content
+  --pf-c-data-toolbar__expandable-content--PaddingTop: var(--pf-c-data-toolbar__content--PaddingTop);
+  --pf-c-data-toolbar__expandable-content--PaddingRight: var(--pf-c-data-toolbar__content--PaddingRight);
+  --pf-c-data-toolbar__expandable-content--PaddingBottom: var(--pf-c-data-toolbar__content--PaddingBottom);
+  --pf-c-data-toolbar__expandable-content--PaddingLeft: var(--pf-c-data-toolbar__content--PaddingLeft);
+  --pf-c-data-toolbar__expandable-content--ZIndex: var(--pf-global--ZIndex--xs);
+  --pf-c-data-toolbar__expandable-content--BoxShadow: var(--pf-global--BoxShadow--md-bottom);
+
+  // Expanded
+  --pf-c-data-toolbar__expandable-content--m-expanded--GridRowGap: var(--pf-global--gutter);
+
+  // Base spacer - shared value
+  --pf-c-data-toolbar--spacer--base: var(--pf-global--spacer--md);
+
+  // Spacer values
+  --pf-c-data-toolbar__item--spacer: var(--pf-c-data-toolbar--spacer--base);
+  --pf-c-data-toolbar__group--spacer: var(--pf-c-data-toolbar--spacer--base);
+
+  // Item
+  --pf-c-data-toolbar--item--Display: flex;
+  --pf-c-data-toolbar--item--Visibility: visible;
+
+  // Toggle group
+  --pf-c-data-toolbar__group--m-toggle-group--spacer: var(--pf-global--spacer--sm);
+  --pf-c-data-toolbar__group--m-toggle-group--m-reveal--spacer: var(--pf-c-data-toolbar--spacer--base);
+  --pf-c-data-toolbar__toggle--c-badge--MarginLeft: var(--pf-global--spacer--sm);
+
+  // Button group
+  --pf-c-data-toolbar__group--m-button-group--spacer: var(--pf-c-data-toolbar--spacer--base);
+  --pf-c-data-toolbar__group--m-button-group--space-items: var(--pf-global--spacer--sm);
+
+  // Icon button group spacer
+  --pf-c-data-toolbar__group--m-icon-button-group--spacer: var(--pf-c-data-toolbar--spacer--base);
+  --pf-c-data-toolbar__group--m-icon-button-group--space-items: 0;
+
+  // Icon group spacer
+  --pf-c-data-toolbar__group--m-filter-group--spacer: var(--pf-c-data-toolbar--spacer--base);
+  --pf-c-data-toolbar__group--m-filter-group--space-items: 0;
```

Comment:
Have you considered applying spacing similar to the input group here so that you don't end up with double borders where elements meet adjacent elements?